src/storage/file_io.h

```diff
@@ -98,27 +97,17 @@ typedef enum
 typedef enum
 {
   FILEIO_ZIP_NONE_METHOD,	/* None */
-  FILEIO_ZIP_LZO1X_METHOD,	/* LZO1X */
+  FILEIO_ZIP_LZ4_METHOD,	/* LZ4X */
```

Comment:
FILEIO_ZIP_LZ4_METHOD enumeration value is the same as FILEIO_ZIP_LZO1X_METHOD.
So, backup header information can have the same value for zip method even though the method is different.
I wonder, for example, a backup volume by FILEIO_ZIP_LZO1X_METHOD from prior version might be misunderstood as FILEIO_ZIP_LZ4_METHOD.

Reply:
I've readded the FILEIO_ZIP_LZO1X_METHOD enum to return an error on trying to restore the old format.